### PR TITLE
How to setup Azure Alerts for Service Fabric Windows node performance counters and How to setup Azure Alerts for Service Fabric Linux node performance counters

### PR DESCRIPTION
**Pull request by jagilber**
new TSG draft to describe how to setup Azure Alerts with WadCfg and performance monitor counters

### CODE REVIEW

File: Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
++                                                    }
++                                                ]
+                                             }
+                                         },
+                                         "StorageAccount": "[parameters('applicationDiagnosticsStorageAccountName')]"
+```
+
+## Updating deployment configuration
+
+If using a template to modify configuration, after .\template.json has been modified, run 'test-azResourceGroupDeployment' and 'new-azResourceGroupDeployment' to deploy new template.
```
Comment (inputoutputcode) [inline Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md:222]: For readability: Test-AzResourceGroupDeployment, New-AzResourceGroupDeployment

File: Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+
+```powershell
+PS C:\>test-azResourceGroupDeployment -resourceGroupName servicefabriccluster -templateFile .\template.json -verbose
```
Comment (inputoutputcode) [inline Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md:225]: Same here Test-AzResourceGroupDeployment -ResourceGroupName servicefabriccluster -TemplateFile .\template.json -Verbose

File: Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md [hunk 3]
```diff
@@ -0,0 +1,460 @@
...
+VERBOSE: 17:58:51 - Template is valid.
+PS C:\>new-azResourceGroupDeployment -resourceGroupName servicefabriccluster -templateFile .\template.json -deploymentDebugLogLevel all -verbose
```
Comment (inputoutputcode) [inline Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md:227]: And here: New-AzResourceGroupDeployment -ResourceGroupName servicefabriccluster -TemplateFile .\template.json -DeploymentDebugLogLevel all -Verbose

Comment (jagilber) [reply Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md]: fixed

File: Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md (hunk 2 above)
Comment (jagilber) [reply Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md]: fixed

File: Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md (hunk 1 above)
Comment (jagilber) [reply Cluster/How to Setup Azure Alerts for Service Fabric Node Performance Counters.md]: fixed

**Pull request merged**